src/modules/facility/facility.service.ts

```diff
@@ -76,16 +76,30 @@ export class FacilityService {
       throw new BadRequestException('Bad request', { description: 'Facility stage code is not issued' });
     }
 
+    const acbsUpdateFacilityRequest: AcbsUpdateFacilityRequest = await this.buildAcbsUpdateFacilityRequest(updateFacilityRequest, facilityIdentifier, idToken);
+    await this.acbsFacilityService.updateFacilityByIdentifier(portfolioIdentifier, acbsUpdateFacilityRequest, idToken);
```

Comment:
Shall we wrap these 2 lines in a function, since we use them in the function below as well?